errs/errors.go

```diff
@@ -180,7 +181,8 @@ func (d *Error) Message() string {
 	default:
 		var buffer strings.Builder
 		buffer.WriteString(fmt.Sprintf("Multiple (%d) errors occurred:", len(d.errors)))
-		for _, one := range d.errors {
+		for i := range d.errors {
+			one := &d.errors[i]
 			buffer.WriteString("\n- ")
 			buffer.WriteString(one.message)
 		}
```

Comment:
Yes, there was at least one case I came across over the years where it was needed. Don't recall what for, though. Rather annoying since the linters will flag the disparity in use (struct vs *struct)... but the same problem exists for all of the marshaling interfaces, too.

Reply:
One good thing though, although it makes an unnecessary copy, 1. non-pointer funcs work on the pointer interface and 2. the struct is fairly small